t/40-script_load_templates.t

```diff
@@ -23,13 +23,32 @@ use Test::More;
 use Test::Output;
 use Test::Warnings;
 use OpenQA::Test::Utils qw(run_cmd test_cmd);
+use Text::Diff 'diff';
+use Mojo::File 'path';
+use Mojo::JSON;    # booleans
+use Cpanel::JSON::XS ();
 
 
 sub test_once {
     # Report failure at the callsite instead of the test function
     local $Test::Builder::Level = $Test::Builder::Level + 1;
     test_cmd(path(curfile->dirname, '../script/load_templates')->realpath, @_);
 }
+sub dump_templates {
+    # Report failure at the callsite instead of the test function
+    local $Test::Builder::Level = $Test::Builder::Level + 1;
+    test_cmd(path(curfile->dirname, '../script/dump_templates')->realpath, @_);
+}
+sub recode {
+    my ($filename) = @_;
+    return Cpanel::JSON::XS->new->relaxed->decode(path($filename)->slurp);
+}
+sub is_samey {
+    # Report failure at the callsite instead of the test function
+    local $Test::Builder::Level = $Test::Builder::Level + 1;
+    my ($a, $b, $msg) = @_;
```

Comment:
I guess `$a` and `$b` can be special variables in Perl so these names shouldn't be used normally. Besides, it looks like these variables are supposed to be file paths. Why not naming them `$file_path_1` and `$file_path_2`?

And by the way, "samey" means "eintönig". I suppose this function does a comparison ignoring the exact format. So what about naming the function `is_yaml_structure_same`?

Reply:
YAML is not what the function is about... but maybe `is_deeply_unordered` is a better name

Reply:
Turns out we don't need the function in the end... having a bit of a dejavu here...